.github/workflows/ci.yml

```diff
@@ -8,37 +8,62 @@ jobs:
       matrix:
-        free-solid-svg-icons: [5.x, 6.x]
-        fontawesome-svg-core: [1.2.x, 6.x]
-        node-version: [18.x, 20.x, 22.x]
-        vue: [3.0.x, 3.1.x, 3.2.x, 3.3.x]
+        free-solid-svg-icons: [7.x, 6.x, 5.x]
+        fontawesome-svg-core: [7.x, 6.x, 1.2.x]
+        node-version: [20.x, 22.x]
+        vue: [3.0.x, 3.1.x, 3.2.x, 3.3.x, 3.4.x, 3.5.x]
+        exclude:
+          # Exclude Font Awesome 7.x free-solid-svg-icons with older svg-core versions
+          - free-solid-svg-icons: '7.x'
+            fontawesome-svg-core: '6.x'
+          - free-solid-svg-icons: '7.x'
+            fontawesome-svg-core: '1.2.x'
 
     steps:
       - uses: actions/checkout@v1
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
-      - name: v5 install, build, and test
-        if: ${{ matrix.free-solid-svg-icons }} == '5.x'
+      - name: v7 install, build, and test
+        if: ${{ matrix.free-solid-svg-icons == '7.x' && matrix.fontawesome-svg-core == '7.x' && matrix.vue >= '3.2.x' }}
         run: |
-          npm install -g npm@8
+          npm install -g npm@10
           npm install
-          npm install --no-save @fortawesome/fontawesome-svg-core@1.2.x @fortawesome/free-solid-svg-icons@5.x vue@${{ matrix.vue }}
+          npm install --no-save ./fortawesome-fontawesome-svg-core-7.0.0-alpha8.tgz ./fortawesome-fontawesome-common-types-7.0.0-alpha8.tgz @fortawesome/free-solid-svg-icons@6.x vue@${{ matrix.vue }}
+          if [[ "${{ matrix.vue }}" < "3.2.13" ]]; then
+            npm install --no-save @vue/server-renderer@${{ matrix.vue }}
+          fi
           npm run build
           npm list vue
           npm run test
       - name: v6 install, build, and test
-        if: ${{ matrix.free-solid-svg-icons }} == '6.x'
+        if: ${{ matrix.free-solid-svg-icons == '6.x' }}
         run: |
-          npm install -g npm@8
+          npm install -g npm@10
           npm install
           npm install --no-save @fortawesome/fontawesome-svg-core@6.x @fortawesome/free-solid-svg-icons@6.x vue@${{ matrix.vue }}
+          if [[ "${{ matrix.vue }}" < "3.2.13" ]]; then
+            npm install --no-save @vue/server-renderer@${{ matrix.vue }}
+          fi
+          npm run build
+          npm list vue
+          npm run test
+      - name: v5 install, build, and test
+        if: ${{ matrix.free-solid-svg-icons == '5.x' }}
+        run: |
+          npm install -g npm@10
+          npm install
+          npm install --no-save @fortawesome/fontawesome-svg-core@1.2.x @fortawesome/free-solid-svg-icons@5.x vue@${{ matrix.vue }}
+          if [[ "${{ matrix.vue }}" < "3.2.13" ]]; then
+            npm install --no-save @vue/server-renderer@${{ matrix.vue }}
+          fi
           npm run build
           npm list vue
           npm run test
         env:
           CI: true
       - name: dist
         run: |
+          npm install
```

Comment:
I was getting cross env CI Test failures.  The solution to the error was to `npm install` prior to `npm run dist`.